README.md

```diff
@@ -1,2 +1,43 @@
 # brandingUpdate
 
+
+The following table provides a summary of the terms that are changing in accordance with the recent branding guidelines.
+
+When directly referenced in the documentation, all of the terms listed below should be updated to their rebranded counterparts as capabilities of Adobe Experience Platform.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+|  **Platform Launch Server Side** |  event forwarding | Launch Server Side is now event forwarding. It is written in lower case as it is a capability not a service.  | See the **Launch Server Side** overview and Getting started with Experience **Platform Launch Server Side** for more information.  | See the **event forwarding overview** and Getting started with **event forwarding** for more information.  |
+| **edge configuration**  |  datastream | The term 'edge configuration' should be replaced with the lowercase noun 'datastream'. It is written in lower case as it is not a branded service. | Configure the Web SDK extension for Platform Launch Client Side to send data to the **edge configuration** that you created previously.  | Configure the Web SDK tag extension to send data to the datastream created previously.  |
+| **Platform Launch**  | tags in Adobe Experience Platform  | If context is required by removing Launch from the sentence, you can append 'in Adobe Experience Platform' on the first instance. Subsequent references can simply refer to the suite of technologies as tags. | What is **Platform Launch?**  | What are **tags in Adobe Experience Platform?** |
+
+The UI in Adobe Experience Platform that provides Platform Launch capabilities has now been renamed as the Data Collection UI. When a concept or capability is described as being "in" Launch, the author is likely referring to the Data Collection UI as it was previously known as the Launch UI. 
+
+Despite "Adobe Experience Platform Data Collection" appearing in the UI, you should avoid having "Adobe Experience Platform" and "data collection" next to each other since it is not a branded service. Data collection is only capitalized in reference to the UI or when it is directly referred to as category, although that is rare. An example of this is provided in the table below.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+| **When configured in Adobe Experience Platform Launch, ...**  | When configured in the Data Collection UI... | When referring to something "in" Launch, it is assumed that the author is referring to the UI.  | **When configured in Adobe Experience Platform Launch**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  | **When configured in the Data Collection UI**, this extension sends AEM Asset Impression and Click metrics to Adobe Analytics.  |
```

Comment:
This "original term" is not a term? It's an example. Either update the term so it's an actual term being replaced, or change the table column labels so this isn't confusing. A lot of this information is just repeating the previous paragraph anyway...we can just pare this down to a set of examples.